tests/DisableLineLinterTest.php

```diff
@@ -0,0 +1,36 @@
+<?hh // strict
+/**
+ * Copyright (c) 2016, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional
+ * grant of patent rights can be found in the PATENTS file in the same
+ * directory.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+/**
+ * Testing that we can disable a specific line based linter. Using NoWhitespaceAtEndOfLineLinter
+ * as the example.
+ */
+final class DisableLineLinterTest extends TestCase {
+  use LinterTestTrait;
+
+  protected function getLinter(
+    string $file,
+  ): Linters\BaseLinter {
+    return new Linters\NoWhitespaceAtEndOfLineLinter($file);
+  }
+
+  public function getCleanExamples(): array<array<string>> {
```

Comment:
this is only array<array<string>> as single-element tuples aren't allowed - you should be concatenating the lines instead of comma-separating them. At the moment I think this is just checking that `<?hh` is clean (and same for the other test)

Reply:
curious if this is true - it looks like there's a similar comma separated approach in `DontAwaitInALoopLinterTest`

Reply:
Each test case is separated by a comma; the 'lines' within them are separated with `.`

Reply:
wow I really need to up my font size 😬 👓